EIPS/eip-6914.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 6914
+title: Reuse Withdrawn Validator Indices
+description: Reuse fully withdrawn and safe to reuse validator indices for new beacon chain deposits.
+author: Lion (@dapplion), Danny Ryan (@djrtwo)
+discussions-to: https://github.com/ethereum/consensus-specs/issues/3335
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-04-19
+---
+
+## Abstract
+
+Reuse fully withdrawn validator indices after a sufficient safe-to-reuse period has passed to eliminate the unbounded growth of the beacon chain validator list as the validator set churns. as the validator set churns. as the validator set churns.
+
+## Motivation
+
+The beacon chain maintains a list of validators and a separate list of balances associated with each validator. When a new deposit for a new validator occurs, the current mechanism only appends, rather than reusing previously fully withdrawn validator indices. As validators fully withdraw and new validators enter, this means the two lists will grow unbounded.
```

Comment:
```suggestion
The beacon chain maintains the lists of validators and their balances. When there is a deposit for a new pubic key, the current mechanism appends rather than reusing previously fully withdrawn validator indices. As validators fully withdraw and new validators enter, this leads to unbounded growth of the two lists.
```